### DISCUSSION TITLE
ZK stack as L3 Rollup

### DISCUSSION THREAD

**Discussion by devudilip** [zkSync-Community-Hub/zksync-developers#503] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement.

Can we use the ZK stack as L3 Rollup, as in transaction will settle on any of the L2?
 example Zk sync, Base chain etc. 
 
While using the zk stack init, I can see the settlement is only for L1(Sopolia/ethereum). And I can run as L2 ZK. 

But I would love to explore more on ZK to see if we have any possibilities to spin up as a Layer3 rollup. 
can

### What is the specific issue or error you're encountering?

No option to use Layer 2 chain as settlement. the 

### Can you share the error messages or logs you're receiving, if any?

There are no erros, but would like to explore more 

### Have you made any recent changes to the contract before encountering this issue?

No

### Are there any external libraries or contracts that your contract interacts with?

NO

### Can you provide the relevant portions of your contract code where the issue is occurring?

NA

### Have you tried to isolate the problem, and if so, what were the results?

NA

### What steps have you already taken to try to resolve the issue?

Tried using the Custom chain as Base chain and ZK sync chain but didn't work

**Comment by bxpana** [comment]
Will double check on this for you.

**Reply by devudilip** [reply]
thank you

**Reply by bxpana** [reply, marked as answer]
We will have options for different DAs in the future, but not sure if it will be to different L2s. Here's the latest update we have on that https://twitter.com/zkSyncDevs/status/1775921157750895033

**Reply by devudilip** [reply]
Yeah, i have seen this. 
But I am curious to know if we can change the settlement layer. So we can use ZK stack to rollup on Zksync

**Reply by bxpana** [reply]
Not in the short term. Right now all ZK Stack chains will be "L2s" that settle to L1. You will have the option to run a Validium with DA not on Ethereum, so that you can have cheaper transactions

**Reply by devudilip** [reply]
Understood, thanks for the help :-)

**Reply by bxpana** [reply]
Got some clarification that we will have the option for a rollup to settle on zkSync Era, but it will not have the full interoperability with the rest of the ZK Stack chains.

**Reply by devudilip** [reply]
ahh that's a piece of great news. 
We want to explore this part.

**Reply by bxpana** [reply]
Great, feel free to send us an email at zkstack@matterlabs.com if you want to talk more to our team about this.
